app/components/content_container_component.rb

```diff
@@ -0,0 +1,16 @@
+class ContentContainerComponent < ViewComponent::Base
+  def initialize(classes: '', data_attributes: {})
+    @classes = classes
+    @data_attributes = data_attributes
+  end
+
+  private
+
+  attr_reader :classes
+
+  def data_attributes
+    @data_attributes.map do |key, value|
+      "data-#{key.to_s.dasherize}=#{value.to_s.dasherize}"
+    end.join(' ')
+  end
```

Comment:
Neat :ok_hand:

Reply:
@CouchofTomato deserves the credit for this one 💪  he implemented it in another component. Gonna do a follow up PR to make it available to all components.